awstesting/mock/server.go

```diff
@@ -0,0 +1,27 @@
+package mock
+
+import (
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+)
+
+const (
+	filepath = "./awstesting/mock/mock_data.json"
+)
+
+// Session is a mock session which is used to hit the mock server
+var Session = session.New(&aws.Config{
+	DisableSSL: aws.Bool(true),
+	Endpoint:   aws.String(server.URL[7:]),
+})
+
+// server is the mock server that simply writes a 200 status back to the client
+var server = httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	resp := http.Response{
+		StatusCode: 200,
+	}
+	resp.Write(w)
```

Comment:
This can be just the follow. Since no other values are being sent don't need to create a Response.

```go
w.WriteHeader(http.StatusOK)
```